icon_validator/rules/plugin_validators/version_pin_validator.py

```diff
@@ -0,0 +1,24 @@
+import re
+import os
+
+from icon_validator.rules.validator import KomandPluginValidator
+from icon_validator.exceptions import ValidationException
+
+
+class VersionPinValidator(KomandPluginValidator):
+
+    @staticmethod
+    def read_requirements(spec):
+        with open(os.path.join(spec.directory, "requirements.txt")) as requirements:
+            return requirements.read().strip()
+
+    def validate(self, spec):
+        requirements_text = self.read_requirements(spec).split("\n")
+        for requirements_text_one_element in requirements_text:
+            if requirements_text_one_element.startswith("#"):
+                continue
+            if not re.match(r'[^=]*?==[^=]*?', requirements_text_one_element):
```

Comment:
We'll need to make sure the regex here validates correctly against the valid version specifiers defined in PEP 508: https://www.python.org/dev/peps/pep-0508/#complete-grammar Under the `version_cmp` property